WinUIGallery/ControlPages/ItemsViewPage.xaml

```diff
@@ -0,0 +1,311 @@
+            </ItemContainer>
+        </DataTemplate>
+    </local:ItemsPageBase.Resources>
+
+    <StackPanel>
+        <local:ControlExample x:Name="Example1" HeaderText="Basic ItemsView">
+            <StackPanel>
+                <TextBlock Margin="0,0,0,15">
+                    This is a basic ItemsView which uses its default StackLayout layout and a simple ItemTemplate.<LineBreak/>Hit the Enter key, double-click or double-tap an item to invoke it.
+                </TextBlock>
+                <ItemsView
+                    x:Name="BasicItemsView"
+                    Width="220" Height="400" HorizontalAlignment="Left"
+                    ItemTemplate="{StaticResource ImageTemplate}"
+                    IsItemInvokedEnabled="True"
+                    ItemInvoked="BasicItemsView_ItemInvoked"/>
+                <TextBlock x:Name="tblBasicInvokeOutput" Style="{StaticResource OutputTextBlockStyle}"/>
+            </StackPanel>
+            
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;ItemsView Width="220" Height="400" HorizontalAlignment="Left"
+    ItemTemplate="{StaticResource ImageTemplate}" IsItemInvokedEnabled="True" ItemInvoked="BasicItemsView_ItemInvoked"/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.CSharp>
+                <x:String xml:space="preserve">
+private void BasicItemsView_ItemInvoked(ItemsView sender, ItemsViewItemInvokedEventArgs e)
+{
+    tblBasicInvokeOutput.Text = "You invoked " + (e.InvokedItem as CustomDataObject).Title + ".";
+}
+                </x:String>
+            </local:ControlExample.CSharp>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example2" HeaderText="ItemsView with swappable layouts">
+            <StackPanel>
+                <TextBlock TextWrapping="Wrap" Margin="0,0,0,15">
+                    Use the options on the right to control different layout customizations to the ItemsView below.
+                </TextBlock>
+                <ItemsView x:Name="SwappableLayoutsItemsView"
+                    Width="500" Height="400" HorizontalAlignment="Left"
+                    ItemTemplate="{StaticResource LinedFlowLayoutItemTemplate}">
+                    <ItemsView.Layout>
+                        <LinedFlowLayout ItemsStretch="Fill" LineHeight="160" LineSpacing="5" MinItemSpacing="5"/>
+                    </ItemsView.Layout>
+                </ItemsView>
+            </StackPanel>
+            
+            <local:ControlExample.Options>
+                <StackPanel MinWidth="300">
+                    <StackPanel.Resources>
+                        <Style x:Key="DefaultNumberBoxStyle" TargetType="NumberBox">
+                            <Setter Property="SpinButtonPlacementMode" Value="Inline"/>
+                            <Setter Property="SmallChange" Value="1"/>
+                            <Setter Property="HorizontalAlignment" Value="Stretch"/>
```

Comment:
This should be `Left` so the NumberBox left aligns.

Reply:
I'm trying Left for both the HorizontalAlignment and HorizontalContentAlignment, but it does not look as good as Stretched:
![image](https://github.com/microsoft/WinUI-Gallery/assets/36547063/5c85857d-c9c6-4db6-ab24-1f483d423506)